### PR TITLE
.coafile: Change glob of ApplyPatchAction

### PR DESCRIPTION
**Pull request by clarammdantas**
Closes https://github.com/coala/coala/issues/5013

### CODE REVIEW

File: coalib/coala_main.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def run_coala(console_printer=None,
             if not autoapply:
                 section['default_actions'] = ''
             elif force_show_patch:
-                section['default_actions'] = '*: ShowPatchAction'
+                section['default_actions'] = '**: ShowPatchAction'
```
Comment (ishanSrt) [inline coalib/coala_main.py]: Don't change this in this issue, I am still discussing how to write a good test for this (for ApplyPatchAction) on my PR. Open another issue for this (as done in https://github.com/coala/coala/issues/4957 ) and then change it as this issue is limited to changing the glob in .coafile.

Comment (ishanSrt) [reply coalib/coala_main.py]: BTW you sure this problem also occurs for ShowPatchAction?

**Comment by ishanSrt** [comment]
So you don't need to change it for ShowPatchAction in .coafile? Is that working fine without it?

**Comment by clarammdantas** [comment]
Yes. It is working fine without it. Thanks for your help :)

**Comment by ishanSrt** [comment]
cc @Makman2

**Comment by Makman2** [comment]
Commit message: I would rather rephrase to
```
.coafile: Change glob of default_actions

... in section [python].

Closes xxxx
```
Currently it sounds a bit like you've modified `ApplyPatchAction.py`^^

**Comment by Makman2** [comment]
unack 8a7460e4ce87d686981905bcd17f1ddaf0b8b418

**Comment by clarammdantas** [comment]
@Makman2 and @ishanSrt it is ok now?

**Comment by Makman2** [comment]
ack 968d696e52430eadb9f92b0f696f708a1bd68b53

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@gitmate-bot ff

**Pull request merged**